### PR TITLE
Remove my name (Test Pull Request)

### PR DESCRIPTION
**Pull request by EdvardasDlugauskas**
Here are my changes, please accept pull request.

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2018 Marijus Laucevicius, Edvardas Dlugauskas
+Copyright (c) 2018 Marijus Laucevicius
```
Comment (marjasi) [inline LICENSE:3]: What is this?

File: LICENSE [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
 
 Permission is hereby granted, free of charge, to any person obtaining a copy
```
Comment (marjasi) [inline LICENSE:5]: What permission?

**Review by marjasi — CHANGES_REQUESTED** [review]
Reviewing the LICENSE file changes.

File: LICENSE (hunk 1 above)
Comment (EdvardasDlugauskas) [reply LICENSE:3]: 100% needed.

Comment (marjasi) [reply LICENSE:3]: OK

**Review by marjasi — APPROVED** [review]

**Pull request closed without merging**